src/webrtc/callFeed.ts

```diff
@@ -129,9 +167,19 @@ export class CallFeed extends TypedEventEmitter<CallFeedEvent, EventHandlerMap>
         this.analyser.fftSize = 512;
         this.analyser.smoothingTimeConstant = 0.1;
 
-        const mediaStreamAudioSourceNode = this.audioContext.createMediaStreamSource(this.stream);
+        this.secondStream = this.stream.clone();
+        const mediaStreamAudioSourceNode = this.audioContext.createMediaStreamSource(this.secondStream);
         mediaStreamAudioSourceNode.connect(this.analyser);
 
+        const streamNode = this.audioContext.createMediaStreamSource(this.stream);
+        this.audioDelay = this.audioContext.createDelay(0.001);
+        const gainNode = this.audioContext.createGain();
```

Comment:
Why do we need a gain node?

Reply:
This was the hard part to understand, as far as I am aware now, if you don't create a sourcenode, then you can leave this out and it will automatically take care of your audio. This means add automatic gain and more.
If you however do it manually, then you have to add a gainnode yourself.
I noticed that since the audio was suddenly different to what it was before.

With the gainnode the audio goes back to normal, although it would be very beneficial if other people could test this out as well, since I only have 1 pair of ears.

Reply:
Ok, that is interesting... Could you please add a comment about this?

(I'll try to test it at some point but lack the time atm)